### DISCUSSION TITLE
markers in vomnibar active tabs selection

### DISCUSSION THREAD

**Discussion by txtyash** [gdh1995/vimium-c#714] [Q&A] [answered]
I have this binding `map t Vomnibar.activateTabSelection currentWindow` that shows me the currently open tabs from the active window.
A marker would be helpful to indicate the currently focused tabs since many times I have many open tabs with similar titles.

**Comment by gdh1995** [comment, marked as answer]
You may watch the "label" part of Vomnibar - a currently active tab is marked with `#(...)` .

![image](https://user-images.githubusercontent.com/5547703/187056594-cd1acc59-3397-496e-a649-8da7c6b26ff9.png)

**Reply by txtyash** [reply]
Could we still have a more vibrant marker for the currently focused tab?

**Reply by gdh1995** [reply]
I'm not interested in this. You may change the size and colors of (all) suggestion labels by adding CSS rules to `Custom CSS for Vimium C UI` by yourself.
